core/templates/domain/exploration/ParamTypeObjectFactory.ts

```diff
@@ -122,9 +121,8 @@ export class ParamTypeObjectFactory {
     }
     // The bracket notation is needed since 'backendName' is a dynamic property
     // and is not defined on 'registry'.
```

Comment:
Could you help me understand this one more? Looking at https://eslint.org/docs/rules/dot-notation#rule-details, this seems like an example of "correct" usage because the property name is a variable.

Reply:
Removed.